apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/internal/listener/ApolloClientMonitorEventListener.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.monitor.internal.listener;
+
+import com.ctrip.framework.apollo.monitor.internal.event.ApolloClientMonitorEvent;
+import com.ctrip.framework.apollo.monitor.internal.model.SampleModel;
+import java.util.List;
+
+/**
+ * @author Rawven
+ */
+public interface ApolloClientMonitorEventListener {
+
+
+  /**
+   * mbean name
+   */
+  String mBeanName();
+
+  /**
+   * is support the event
+   */
+  boolean isSupport(ApolloClientMonitorEvent event);
+
+  /**
+   * collect metrics from event
+   */
+  void collect(ApolloClientMonitorEvent event);
```

Comment:
see org.springframework.context.ApplicationListener

method name change to `onApolloClientMonitorEvent` will be better?

>